lang/en.yml

```diff
@@ -232,41 +186,45 @@ en:
     TWODIGITDAY: 'Two-digit day of month'
     TWODIGITYEAR: 'Two-digit year'
     Toggle: 'Show formatting help'
-  MoneyField:
-    FIELDLABELAMOUNT: Amount
-    FIELDLABELCURRENCY: Currency
-  MultiSelectField:
+  SilverStripe\Security\MemberLoginForm:
+    AUTHENTICATORNAME: 'E-mail &amp; Password'
+  SilverStripe\Forms\MoneyField:
+    INVALID_CURRENCY: 'Currency {currency} is not in the list of allowed currencies'
+  SilverStripe\Forms\MultiSelectField:
     SOURCE_VALIDATION: 'Please select values within the list provided. Invalid option(s) {value} given'
-  NullableField:
+  SilverStripe\Forms\NullableField:
     IsNullLabel: 'Is Null'
-  NumericField:
+  SilverStripe\Forms\NumericField:
     VALIDATION: '''{value}'' is not a number, only numbers can be accepted for this field'
-  Pagination:
+  SilverStripe\Forms\GridField\GridFieldPaginator_Row.ss:
     Page: Page
     View: View
-  PasswordValidator:
+  SilverStripe\Security\PasswordValidator:
     LOWCHARSTRENGTH: 'Please increase password strength by adding some of the following characters: %s'
     PREVPASSWORD: 'You''ve already used that password in the past, please choose a new password'
     TOOSHORT: 'Password is too short, it must be %s or more characters long'
-  Permission:
-    AdminGroup: Administrator
-    FULLADMINRIGHTS: 'Full administrative rights'
-    FULLADMINRIGHTS_HELP: 'Implies and overrules all other assigned permissions.'
```

Comment:
Something broke... these keys got lost.
It's probably because we merged `Permission` with `Permissions` and only one got saved. Can you manually merge these sorry?